docs/dev-guide/iframe-commands.md

```diff
@@ -631,3 +631,59 @@ api.executeCommand('hideNotification',
     uid: String // Unique identifier for the notification to be removed.
 );
 ```
+
+### roomsInfo
+
+Provides list of available rooms and details of it:
+- `isMainRoom` (true,false), `id`, `jid`
+- participants: `Participant[]`
+    - `id`
+    - `jid`
+    - `role`
+    - `displayName`
+
+```javascript
+api.executeCommand('rooms-info');
```

Comment:
Just as a note for future reference, the commands are using the value from the external api commands map, so here we should pass the camel case one.